index.php

```diff
@@ -1,70 +1,20 @@
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Bike Collector</title>
-    <link rel="stylesheet" href="styles/styles.css">
-    <link href="https://fonts.googleapis.com/css?family=Martel:200,300,400,600,700,800,900|Poppins:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&display=swap" rel="stylesheet">
-</head>
+<?php
+    require 'includes/header.php';
+    require 'includes/db.php';
+    require 'includes/displayBikes.php';
+    require 'includes/deleteBike.php';
+?>
+
 <body>
     <h1>Bike Collector App</h1>
     <div class='card-container'>
-        <div class='card'>
-            <img class='bike-image' src="images/sb130.jpeg" alt="Yeti sb130 bike">
-            <div class="card-info-container">
-                <section>Make: Yeti</section>
-                <section>Model: sb130</section>
-                <section>Discipline: Trail</section>
-                <section>Wheel Size: 29er</section>
-            </div>
-            <div class='card-button-container'>
-                <button>Edit Bike</button>
-                <button>Delete Bike</button>
-            </div>
-        </div>
-        <div class='card'>
-        <img class='bike-image' src="images/sb130.jpeg" alt="Yeti sb130 bike">
-            <div class="card-info-container">
-                <section>Make: Yeti</section>
-                <section>Model: sb130</section>
-                <section>Discipline: Trail</section>
-                <section>Wheel Size: 29er</section>
-            </div>
-            <div class='card-button-container'>
-                <button>Edit Bike</button>
-                <button>Delete Bike</button>
-            </div>
-        </div>
-        <div class='card'>
-        <img class='bike-image' src="images/sb130.jpeg" alt="Yeti sb130 bike">
-            <div class="card-info-container">
-                <section>Make: Yeti</section>
-                <section>Model: sb130</section>
-                <section>Discipline: Trail</section>
-                <section>Wheel Size: 29er</section>
-            </div>
-            <div class='card-button-container'>
-                <button>Edit Bike</button>
-                <button>Delete Bike</button>
-            </div>
-        </div>
-        <div class='card'>
-        <img class='bike-image' src="images/sb130.jpeg" alt="Yeti sb130 bike">
-            <div class="card-info-container">
-                <section>Make: Yeti</section>
-                <section>Model: sb130</section>
-                <section>Discipline: Trail</section>
-                <section>Wheel Size: 29er</section>
-            </div>
-            <div class='card-button-container'>
-                <button>Edit Bike</button>
-                <button>Delete Bike</button>
-            </div>
-        </div>
+        <?php  
+            $bikes = getBikesFromDB($db);
+            echo printCards($bikes);
```

Comment:
What if there are no bikes found?

Reply:
it does the function but nothing happens